src/components/organisms/Nav.js

```diff
@@ -37,11 +47,49 @@ const Nav = () => {
    * @param {Object} event
    */
   const handleSignup = (event) => {
-    const { email, password } = event.target.elements;
-    signup(email.value, password.value);
+    const { name, email, password } = event.target.elements;
+    signup(name.value, email.value, password.value);
```

Comment:
this is actually a firebase call to the user signup method. this method strictly only takes an email/password. leave this function alone for now
```suggestion
    const { email, password } = event.target.elements;
    signup(email.value, password.value);
```